cmd/ttn-lw-cli/commands/simulate.go

```diff
@@ -614,5 +621,6 @@ func init() {
 
 	simulateCommand.AddCommand(simulateDataUplinkCommand)
 
+	simulateCommand.PersistentFlags().String("gateway-api-key", "", "API key used for linking the gateway (optional if the cluster allows unregistered gateways)")
```

Comment:
Otherwise the CLI just uses the user's auth (which likely has LINK rights for the gateway)